core/templates/pages/story-editor-page/editor-tab/story-editor.directive.ts

```diff
@@ -58,6 +60,8 @@ angular.module('oppia').directive('storyEditor', [
             $scope.disconnectedNodes = [];
             $scope.linearNodesList = [];
             $scope.nodes = [];
+            $scope.allowedColors = [
+              '#F8BF74', '#D68F78', '#8EBBB6', '#B3D8F1'];
```

Comment:
Just flagging this so that it shows up in the latest set of review comments.

I'm a bit confused about the "global variable" bit. Is it because you already have [this](https://github.com/oppia/oppia/blob/cdec6e81cac46c2ef75ca072d9a188116175f47a/core/templates/pages/story-editor-page/editor-tab/story-editor.directive.ts#L30)? I'll assign @bansalnitish to this PR so he can advise; you might want to ping him too.

Reply:
I'm not getting why we cannot declare this in constants.ts?

Reply:
I have declared it in constants.ts but I'm unable to require and assign it to a variable called constants here because of a global variable conflict. This seems to work fine in the migrated files but does not work here. I think the exact error i got was "Cannot redeclare block-scoped variable 'constants'."

Reply:
Ah I see. I don't think this would work in AngularJS then. Can we replace the variable name from constants to something else and add a comment saying when this file migrates to Angular change this variable name to constants? What are your thoughts?

Reply:
Yeah, I think that sounds reasonable. Thanks @bansalnitish!